app/src/main/java/org/projectbuendia/client/ui/chart/PatientChartActivity.java

```diff
@@ -420,30 +386,6 @@ private String getFormattedPcrString(double pcrValue) {
         }
 
         @Override public void updatePatientConditionUi(String generalConditionUuid) {
```

Comment:
Quick note: all the code related to (a) selecting the general condition with a specialized dialog and (b) tapping a tile to navigate directly to that section of the form was commented out but left in the code because it is functionality that the Ebola deployment depended on.  When we switched to WebView rendering, this code no longer compiled, as it was written for the original Android widget layouts.  It is commented out to show that although it is currently never called, the removal is temporary in the sense that without this code, the Ebola case suffers a regression (while worth the gains from using a WebView) by losing functionality for which we never made a product decision that it was okay to remove.